### PR TITLE
Update japanese translation file

### PR DESCRIPTION
**Pull request by motofy**
Update japanese translation file.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://motofy-locales-ja.fatfreecrm-fat-free-crm-1bbda9a74e1b4e1e2ffd.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://motofy-locales-ja.fatfreecrm-fat-free-crm-1bbda9a74e1b4e1e2ffd.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: config/locales/ja_fat_free_crm.yml [hunk 1]
```diff
@@ -527,25 +722,81 @@ ja:
...
+  #----------------------------------------------------------------------------
   pluralize:
     comment:
       one: 1 件のコメント
       other: ! '%{count} 件のコメント'
     contact:
-      one: 1 contact
-      other: ! '%{count} contacts'
+      one: 1 件のコメント
+      other: ! '%{count} 件のコメント'
```
Comment (stanleylhs) [inline config/locales/ja_fat_free_crm.yml]: 取引先?

File: config/locales/ja_fat_free_crm.yml [hunk 2]
```diff
@@ -527,25 +722,81 @@ ja:
...
     day:
-      one: 1 日
-      other: ! '%{count} 日'
+      one: 1 件のコメント
+      other: ! '%{count} 件のコメント'
     login:
-      one: 1 login
-      other: ! '%{count} logins'
+      one: 1 件のコメント
+      other: ! '%{count} 件のコメント'
```
Comment (stanleylhs) [inline config/locales/ja_fat_free_crm.yml]: All these lines 783 to 796 are wrong?

**Comment by stanleylhs** [comment]
👍 Boots up alright. Commit it for now, let's fix the error later?

**Comment by johnnyshields** [comment]
@CloCkWeRX I'll have my team review this when we roll this out in Japanese. We are based in Japan.

**Comment by johnnyshields** [comment]
Replaced by #693

**Pull request closed without merging**